src/utils/web.ts

```diff
@@ -33,15 +34,15 @@ export async function fetchJson(connection: string | ConnectionInfo, json?: stri
             });
             if (!response.ok) {
                 if (response.status === 503) {
-                    console.warn(`Retrying HTTP request for ${url} as it's not available now`);
+                    getLogger().debug(`Retrying HTTP request for ${url} as it's not available now`);
                     return null;
                 }
                 throw createError(response.status, await response.text());
             }
             return response;
         } catch (error) {
             if (error.toString().includes('FetchError') || error.toString().includes('Failed to fetch')) {
-                console.warn(`Retrying HTTP request for ${url} because of error: ${error}`);
+                getLogger().debug(`Retrying HTTP request for ${url} because of error: ${error}`);
```

Comment:
This one too looks like a warning.

Reply:
Same as above. Because it automatically retries it seems like a debug message. Then if after the retries it fails it should log an error message